docs/opentracing.asciidoc

```diff
@@ -0,0 +1,97 @@
+[[opentracing]]
+== OpenTracing
+
+Elastic APM RUM agent provides an implementation for the https://opentracing.io/[OpenTracing API].
+The `ElasticApmTracer` translates calls to OpenTracing APIs to Elastic APM and thus allows for reusing existing instrumentation.
+
+The first span of a service will be converted to an Elastic APM `Transaction`,
+subsequent spans are mapped to Elastic APM `Span`.
+
+
+[float]
+[[opentracing-getting-started]]
+=== Getting started
+
+In order to use OpenTracing API with Elastic APM, you need to first create a Tracer:
+
+[source,js]
+----
+const { init: initApm, createTracer } = require('elastic-apm-js-base/dist/bundles/elastic-apm-opentracing.umd.js')
+const elasticApm = initApm({
+    // ...
+})
+const elasticTracer = createTracer(elasticApm)
+----
+
+At this point you can either use `elasticTracer` directly, or register it as a global tracer:
+
+[source,js]
+----
+const opentracing = require('opentracing-javascript')
+opentracing.initGlobalTracer(elasticTracer);
+
+// ... 
+
+const tracer = opentracing.globalTracer();
+----
+
+Please refer to https://github.com/opentracing/opentracing-javascript/[opentracing-javascript] documentation for more detail 
+on OpenTracing API.
+
+NOTE: If you are loading the RUM agent using a script tag, please make sure to use our OpenTracing bundle 
+(`elastic-apm-js-base/dist/bundles/elastic-apm-opentracing.umd.js`) in your scripts instead, then you can create a tracer
+using `window.elasticApm.createTracer()`.
+
+
+[float]
+[[opentracing-apm-tags]]
+=== Elastic APM specific tags
+
+Elastic APM defines some tags which are not included in the OpenTracing specification,
+but are relevant in the context of Elastic APM.
+
+- `type` - sets the type of the transaction or span, e.g. "page-load", or "ext.http".
+           If `type` is not specified, `custom` would be set instead.
+
+
+The following tags are relevant only to the root spans, which are
+translated to Elastic APM transactions:
```

Comment:
Let's move the "where they appear" sentence up here so we don't have to repeat it
```suggestion
translated to Elastic APM transactions.
These tags appear in the "User" tab on the transaction details page in the Elastic APM UI.
```